infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -281,24 +274,22 @@
 
     - name: check if containerized osds are already running
       shell: |
-        docker ps | grep -sq {{ item | regex_replace('/', '') }}
+        docker ps | grep -sqE ceph-osd-[0-9]
       changed_when: false
       failed_when: false
-      with_items: "{{ devices }}"
       register: osd_running
 
-    - name: resolve device(s) path(s)
-      command: readlink -f {{ item }}
+    - name: get osd directories
+      command: ls -1 /var/lib/ceph/osd
+      register: osd_dirs
       changed_when: false
-      with_items: "{{ devices }}"
-      register: devices_canonicalize
 
     - name: unmount all the osd directories
-      command: umount "{{ item.0.stdout }}"1
+      command: umount "{{ item.0.stdout }}"
```

Comment:
taking my comment in consideration for L292, I think you just need `{{ item.0 }}`
+ you will need to prefix your `umount` command with `/var/lib/ceph/osd/` I think